### PR TITLE
openstack-new: add option for security groups

### PR DESCRIPTION
**Pull request by nikromen**

### CODE REVIEW

File: resalloc_openstack/new/main.py [hunk 1]
```diff
@@ -80,6 +93,7 @@ def main():
             args.flavor,
             key_name=key,
             nics=nics,
+            security_groups=_get_security_groups(args),
```
Comment (praiskup) [inline resalloc_openstack/new/main.py]: Isn't `security_groups=args.security_groups or None` equivalent?

File: resalloc_openstack/new/arg_parser.py [hunk 2]
```diff
@@ -90,3 +90,11 @@
     help="comma-separated, key=value,foo=baz arguments (see "
          "nova.servers.create(nics=...) help",
 )
+
+parser.add_argument(
+    "--security-group",
+    action="append",
+    dest="security_groups",
+    help="A comma separated list of security group names. E.g. copr,something,else",
```
Comment (praiskup) [inline resalloc_openstack/new/arg_parser.py]: not a comma separated list now

Comment (nikromen) [reply resalloc_openstack/new/arg_parser.py]: oh, i forgot to remove this from rebase

**Comment by praiskup** [comment]
Thank you!

**Review by praiskup — APPROVED** [review]

**Pull request merged**